routes/adminapi.js

```diff
@@ -54,29 +54,33 @@ router.get('/search', (req, response, next) => {
 /*
 curl -H "Content-Type: application/json" -H "Accept: application/json" \
 -X POST \
+-H "Cookie: userid=123" \
 -d '{"word":"KIng","images":["URL1", "URL2"]}' http://localhost:3000/adminapi/saveword
 */
 router.post('/saveword', function(req, res) {
-  if (config.SAVEWORD_KEY_VALIDATION) {
-    if (!(config.COOKIE_NAME in req.cookies))
-      return res.status(400).send('Cookie not provided.');
-    var sessionId = req.cookies[config.COOKIE_NAME];
-    // if sessionId is not whitelisted, throw error
-  }
-  let status = {
-    'Saved': 200,
-    'Duplicate': 400,
-    'Error': 500,
-  };
-  // validate(req.body)
-  db.checkWordExists(req.body.word, (message) => {
-    if (message === false) {
-      db.insertWordInCollection(req.body, (message) => {
+  if (!(config.COOKIE_NAME in req.cookies))
+  return res.status(400).send('Cookie not provided.');
+  var sessionId = req.cookies[config.COOKIE_NAME];
```

Comment:
just a note:
bro let/const are the new age var..
they also give your lexical scope vs functional scope of var